lib/internal/crypto/hkdf.js

```diff
@@ -110,7 +110,15 @@ function hkdf(hash, key, salt, info, length, callback) {
 
   validateFunction(callback, 'callback');
 
-  const job = new HKDFJob(kCryptoJobAsync, hash, key, salt, info, length);
+  const job = new KDFJob(kCryptoJobAsync,
+    "HKDF",
+    {
+      digest: hash,
```

Comment:
Would it be OK to rename the parameters to write `digest,` here? I tried to not change the public interface at all